src/tools/xlcpp.jam

```diff
@@ -0,0 +1,150 @@
+# Copyright Vladimir Prus 2004.
+# Copyright Toon Knapen 2004.
```

Comment:
Can I just remove those lines?   I notice clang.hpp has no copyright other than

# Distributed under the Boost Software License, Version 1.0.
# (See accompanying file LICENSE_1_0.txt
# or copy at http://www.boost.org/LICENSE_1_0.txt)

I just copied and updated from vacpp.hpp and I have no idea who the original author is for that file.